pom.xml

```diff
@@ -24,14 +24,14 @@
     </organization>
 
     <distributionManagement>
-        <snapshotRepository>
-            <id>ossrh</id>
-            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
-        </snapshotRepository>
         <repository>
-            <id>ossrh</id>
-            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+            <id>womply</id>
+            <url>https://womply.artifactoryonline.com/womply/libs-release-local</url>
         </repository>
+        <snapshotRepository>
+            <id>womply</id>
+            <url>https://womply.artifactoryonline.com/womply/libs-snapshot-local</url>
```

Comment:
do we want to pull the womply repo / version stuff into a separate commit for easier PR-ing to upstream?